LogMonitor/src/LogMonitor/EtwMonitor.cpp

```diff
@@ -30,9 +30,11 @@ static const std::wstring g_sessionName = L"Log Monitor ETW Session";
 
 EtwMonitor::EtwMonitor(
     _In_ const std::vector<ETWProvider>& Providers,
-    _In_ bool EventFormatMultiLine
+    _In_ bool EventFormatMultiLine,

```

Comment:
Do we need multiline anymore?

Reply:
Resolved. Funny thing is this was working before the change even with this line in Main.cpp:
`g_etwMon = make_unique<EtwMonitor>(etwProviders, etwMonMultiLine, logFormat);`
line 188